src/RA_Core.cpp

```diff
@@ -752,32 +752,33 @@ API HMENU CCONV _RA_CreatePopupMenu()
 	HMENU hRA_LB = CreatePopupMenu();
 	if (RAUsers::LocalUser().IsLoggedIn())
 	{
+		// TODO: Use the literals once PR 23 is accepted
 		AppendMenu(hRA, MF_STRING, IDM_RA_FILES_LOGOUT, TEXT("Log&out"));
-		AppendMenu(hRA, MF_SEPARATOR, NULL, NULL);
+		AppendMenu(hRA, MF_SEPARATOR, UINT_PTR{}, nullptr);
```

Comment:
I think it would be more standard/common to use `0` instead of `UINT_PTR{}`.

Reply:
Well I was going to use my literals in a different pull request but felt that I shouldn't copy and paste them. <code>UINT_PTR</code> is basically the same as <code>std::size_t</code>. Starting with C++11, rvalue implicit conversions are flagged.

But I know what you mean.

Reply:
I was looking into it further and <code>UINT_PTR</code> is a variable/polymorphic type, it's either <code>uintptr_t</code>, <code>unsigned int</code>, or <code>unsigned __int64</code> depending on where you got it from. So I believe it's more appropriate to hold off on it for correctness. The other standard way to do it would be <code>UINT_PTR{0}</code> but that's pretty much the same thing. <code>type{}</code> just means <code>default</code>.